### ISSUE TITLE
Wrong normalization in rho plotting?

### ISSUE THREAD

**Issue by lorenzo-zuccato** [ocbe-uio/BayesMallows#342]
Hi, 
I wanted to ask clarifications about the rho plotting from the function plot.BayesMallows. I was perplexed about the normalization in line 69 of the file plot.BayesMallows.R. In case of multiple clusters I think the results does not sum to 1 for every cluster. I would use something like 
```r
df$pct <- ave(df$n, df$cluster, df$item, FUN = function(x){x / sum(x)})
``` 
instead of
```r
df$pct <- df$n / sum(df$n)
``` 
Hope that this makes sense :)

**Comment by wleoncio** [comment]
It would be useful to get a reproducible example for the case of multiple clusters. I've probably misunderstood, but at least in the 2 examples in the `test-plot.R` file the current code adds to 1 and the proposal doesn't (see images below, where a breakpoint was added just before that line).

![bilde](https://github.com/ocbe-uio/BayesMallows/assets/8234768/3c1059f4-700e-4766-900b-ca02e2203d3d)
(...)
![bilde](https://github.com/ocbe-uio/BayesMallows/assets/8234768/b5fdcf05-921e-4983-b6e5-fe9f4e65f9ab)

Lorenzo, can you confirm you're running the latest build (currently, 1.4.0.9002) of the package?

**Comment by wleoncio** [comment]
Thanks for the clarification! This goes over my head, so I'll leave it to @osorensen to analyse the case. :)

**Comment by osorensen** [comment]
Thanks @lorenzo-zuccato and @wleoncio. I'll take a look.

**Comment by osorensen** [comment]
Issue fixed now. Thanks for reporting @lorenzo-zuccato. It actually turned out the ggplot2 could do much of the data processing job, so was able to remove a couple of codelines at the same time.

Note that the plot design is different now, since I'm using `ggplot2::geom_histogram()`, but since a ggplot object is returned, it's possible to define exactly the style you like.

``` r
library(BayesMallows)
set.seed(1234)
bm <- compute_mallows(potato_visual, n_clusters = 3, nmc = 1000)

bm$burnin <- 500

plot(bm, parameter = "rho", item = 1)
```

![](https://i.imgur.com/KjGAgaH.png)

<sup>Created on 2023-11-23 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

**Comment by osorensen** [comment]
Closed by #343
